service/src/SysfsDriver.hpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
```

Comment:
```suggestion
 * Copyright (c) 2015 - 2023, Intel Corporation
```
Please check all the license headers for this.